kernel/Base/Edge.h

```diff
@@ -79,15 +80,16 @@ class Edge {
         return numberOfConformingDOFOnTheEdge_[0];
     }
 
-    std::size_t getLocalNumberOfBasisFunctions(std::size_t unknown) const {
+    std::size_t getLocalNumberOfBasisFunctions(
+        std::size_t unknown) const final {
         // TODO: LC, numberOfConformingDOFOnTheNode_ might be smaller than
```

Comment:
is this still a valid comment?

Reply:
Looks like it already works for face/node, so I doubt it is still up to date.